guacamole-ext/src/main/java/org/apache/guacamole/environment/LocalEnvironment.java

```diff
@@ -27,6 +27,8 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Properties;
+
+import org.apache.guacamole.properties.BooleanGuacamoleProperty;
```

Comment:
Minor style detail - in general I think we try to keep the import statements groups together and in mostly alphabetical order, and we don't tend to put spaces in between different groups of imports.  I don't have a preference one way or the other, but, for consistency's sake, seems like we should stick to that, here.

Reply:
Still trying to tune my style setup for this project. Thanks for catching that.